docs/src/content/docs/de/reference/icons.mdx

```diff
@@ -1,6 +1,8 @@
 ---
 title: Symbole Referenz
 description: Eine Übersicht über alle in Starlight verfügbaren Symbole.
+sidebar:
+  label: Symbole
```

Comment:
Related disccussion for reference https://github.com/withastro/starlight/pull/2470/files/2b486d1b1fa4acf3f0c87da750f3ce219794f78e#diff-9185fd84aba7e9a14b394be96e2bd3562baac10c0285be708a3109c5cbe72802